ui/src/components/Sidebar/Sidebar.tsx

```diff
@@ -181,15 +190,21 @@ export default function SideBar({
               variant="outlined"
               size="small"
               value={fromTimestamp}
-              onChange={(e) => setFromTimestamp(e.target.value)}
+              onChange={(e) => {
+                setFromTimestamp(e.target.value);
+                setValidFromTimestamp(checkValidity(e.target.value) ? e.target.value : '');
+              }}
               error={Number.isNaN(Date.parse(fromTimestamp || '0'))}
```

Comment:
You can now resuse the `checkValidity` function here too:
```jsx
error={!checkValidity(fromTimestamp || '0')}
```

I think a better name for the function would be `isTimestampValid`
- It is declarative and tells you what the function is checking
- Functions that return a boolean are often named this way; you can easily deduce what the function does